torch/_torch_docs.py

```diff
@@ -3932,6 +3932,38 @@
 
 """)
 
+add_docstr(torch._C.set_flush_denormal,
+           r"""
+set_flush_denormal(on) -> bool
+
+Returns ``True`` if it successfully configures flush denormal mode.
```

Comment:
This sounds as if `False` meant that it's a failure to set it, while what it really means is that the particular CPU architecture architecture doesn't support those flags. Can you please move the sentence that says it only works on those CPUs that have SSE3+ here?

Reply:
Done